### PR TITLE
App theme refactor

### PR DESCRIPTION
**Pull request by yunyizhu1**
Refactored App:
[x] chorestar
[x] easypick
[x] groceryship
[x] lingua
[x] listify (1)
[x] livescorecard
[x] phoenix
[x] potluck
[x] rendezvous
[x] sweetspot (2)

Bugs: 
(1) In Listify, the button color is set as `currentColor`, which is determined by color of the surroundings. After using the theme, the surrounding color is changed and I haven't thought of a way to fix it yet. 

(2) The original app almost randomly uses the Roboto and Apple-system fonts. I don't think it's intentional so I used Roboto everywhere.

### CODE REVIEW

File: packages/themes/scss/baby-blue.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// Commented out because the original css didn't import
+// and was showing sans-serif while having 'Roboto'
+// listed as the preferred font
+//@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
```
[the remark this answers is not in the document]
Comment (yunyizhu1) [reply packages/themes/scss/baby-blue.scss]: Resolved!

File: samples/groceryship/src/styles.scss [hunk 2]
```diff
@@ -1,62 +1,21 @@
-@import "~css-star-rating/dist/css/star-rating.min.css";
-@import "~@angular/material/prebuilt-themes/indigo-pink.css";
+
```
[the remark this answers is not in the document]
Comment (yunyizhu1) [reply samples/groceryship/src/styles.scss]: Resolved!

File: samples/groceryship/src/styles.scss [hunk 3]
```diff
@@ -1,62 +1,21 @@
...
+@import "~@deja-vu/themes/scss/vivid-green.scss";
 @import "~bootstrap/dist/css/bootstrap.min.css";
```
[the remark this answers is not in the document]
Comment (yunyizhu1) [reply samples/groceryship/src/styles.scss]: Technically it is not allowed, but I found a workaround [here](https://github.com/sass/sass/issues/451#issuecomment-359489037). Implemented in the newest commit.

File: packages/themes/scss/basil-green.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
+@import url("https://fonts.googleapis.com/css?family=Pacifico");
+@import url("https://fonts.googleapis.com/css?family=Roboto");
```
Comment (spderosso) [inline packages/themes/scss/basil-green.scss]: you should add `&display=swap` to the url (e.g., see the Robot URL in baby-blue)

File: packages/themes/scss/basil-green.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
+@import url("https://fonts.googleapis.com/css?family=Pacifico");
```
Comment (spderosso) [inline packages/themes/scss/basil-green.scss]: move this to potluck's CSS? Pacifico is not used in the theme. Also, add `&display=swap` to the end of the URL

File: packages/themes/scss/bright-blue.scss [hunk 6]
```diff
@@ -0,0 +1,59 @@
+// Commented out because the original css didn't import
```
Comment (spderosso) [inline packages/themes/scss/bright-blue.scss]: Let's go ahead and use Lato, it doesn't matter if the student implementation had a bug and wasn't really using it.

File: packages/themes/scss/soft-green.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
+@import url('https://fonts.googleapis.com/css?family=Asap');
```
Comment (spderosso) [inline packages/themes/scss/soft-green.scss]: add `&display=swap` to the url

File: packages/themes/scss/stylish-pink.scss [hunk 8]
```diff
@@ -0,0 +1,24 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway');
```
Comment (spderosso) [inline packages/themes/scss/stylish-pink.scss]: add `&display=swap`

Comment (yunyizhu1) [reply packages/themes/scss/stylish-pink.scss]: Resolved!

File: packages/themes/scss/soft-green.scss (hunk 7 above)
Comment (yunyizhu1) [reply packages/themes/scss/soft-green.scss]: Resolved!

File: packages/themes/scss/bright-blue.scss (hunk 6 above)
Comment (yunyizhu1) [reply packages/themes/scss/bright-blue.scss]: Resolved!

File: packages/themes/scss/basil-green.scss (hunk 5 above)
Comment (yunyizhu1) [reply packages/themes/scss/basil-green.scss]: Resolved!

File: packages/themes/scss/basil-green.scss (hunk 4 above)
Comment (yunyizhu1) [reply packages/themes/scss/basil-green.scss]: Resolved!

**Comment by spderosso** [comment]
I tested the following apps so far:
- sn: OK
- chorestar: OK
- groceryship: OK. I fixed and pushed to this PR a change to make claim request reactive
- listify: OK, but the welcome text was black instead of white. I pushed the fix to this PR
- mapcampus: OK
- phoenix: the phoenix logo doesn't show up
- rendezvous: map doesn't show up

I still have to test the others, but can you look into phoenix and rendezvous and the other apps I haven't tested so far and make sure that there are no regressions? At this point the changes in `master` are very stable so we should try to avoid merging code that will break things.

**Pull request merged**